components/event-gateway/internal/events/bus/process.go

```diff
@@ -0,0 +1,54 @@
+package bus
+
+import (
+	"fmt"
+
+	"github.com/kyma-project/kyma/components/event-gateway/internal/events/api"
+)
+
+type configurationData struct {
+	sourceNamespace   string
+	sourceType        string
+	sourceEnvironment string
+}
+
+var conf *configurationData
```

Comment:
I would avoid global variables. Consider declaring struct (e.g. EventBus) with configuration and eventTargetURL properties. Init function would be a factory method and return the struct (or an interface if you would like to mock it). What do you think?

Reply:
This is a code copy from the `gateway` component.